src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -511,12 +511,12 @@ static void Main(string[] args)
             var expectedItems = new[]
             {
                 "Rename 'P2' to 'Foober'",
+                "Goober - using N;",
```

Comment:
Why is this test changing? The new behavior looks correct...was the old behavior wrong before?

Reply:
Async lightbulbs change this.  Add-using is in the high-pri bucket.  So its fixes come before the 'generate type' fixes now.

Reply:
@CyrusNajmabadi Why wasn't it like that before though? Add using was always high priority.